cli/running/lua/launcher.lua

```diff
@@ -49,18 +203,16 @@ local function cfg_wrapper(cfg)
     ffi.C.chdir(os.getenv('TT_CLI_CONSOLE_SOCKET_DIR'))
     local cfg = cfg or {}
     local tt_cfg = {}
-    tt_cfg.wal_dir = os.getenv('TT_WAL_DIR')
-    tt_cfg.memtx_dir = os.getenv('TT_MEMTX_DIR')
-    tt_cfg.vinyl_dir = os.getenv('TT_VINYL_DIR')
+    tt_cfg = get_env_cfg()
     for i, v in pairs(tt_cfg) do
         if cfg[i] == nil then
             cfg[i] = v
         end
     end
     local success, data = pcall(origin_cfg, cfg)
     ffi.C.chdir(os.getenv('TT_CLI_WORK_DIR'))
-    if not success then
-        log.error('Someting wrong happened when tried to set dataDir.')
+    if not success then 
```

Comment:
![image](https://user-images.githubusercontent.com/57662928/207840729-d6feb314-da00-4685-b6dd-9813f46c1ba5.png)
Remove space at the end of the string.